### ISSUE TITLE
Running should change the FOV

### ISSUE THREAD

**Issue by leorockway** [Wuzzy2/MineClone2-Bugs#209]
I'm pretty sure this isn't possible with the current engine, but running should change the field of view as a visual indication of running.

**Comment by Wuzzy2** [comment]
You're right, this is definitely an engine feature. I strongly suggest you open an issue for Minetest as well (if there isn't already one). There is pretty much nothing I can do about this.

Fun fact: Freeminer already has this.

**Comment by leorockway** [comment]
I remember trying a fork a few years ago that had this. It wasn't called Freeminer at the time, so either that got renamed Freeminer or Freeminer got the code from this other project (I think it was called Blockworld). But, like I said, this was a few years ago, so it's kind of sad that MT still doesn't have it. I'll look at the MT issues and I'll open one if I don't see it there.

**Comment by leorockway** [comment]
Relevant issue for Minetest: https://github.com/minetest/minetest/issues/3165

**Comment by tacotexmex** [comment]
https://github.com/minetest/minetest/commit/f470cb7270560a26afff0a0eadfc69070236c109
